fda-model/metrics/AlternativeComparisonReportResults.cs

```diff
@@ -89,6 +106,27 @@ public void ReportMessage(object sender, MessageEventArgs e)
         {
             MessageReport?.Invoke(sender, e);
         }
+
+        public XElement WriteToXML()
+        {
+            XElement mainElement = new XElement("AlternativeComparisonReportResults");
```

Comment:
For writing and reading from xml, it is best to use string constants so that we can update it in one place instead of having to change it in the write and the read. 
example:        public const string STAGE_DAMAGE_ID = "StageDamageID";

This is a general rule for any string that will be used in more than one place.

Reply:
I created a ticket to do this for all hard coded stuff